examples/service.json

```diff
@@ -0,0 +1,6 @@
+{
+  "service": "python3 -m http.server --bind 127.0.0.1",
```

Comment:
I'm assuming the 8000 is implicit here but it might be better to be explicit with the port?

Reply:
8000 is always implicit with `http.server`, yeah.